backend/src/v4/routes/teamspace.js

```diff
@@ -547,18 +547,17 @@
 	}
 
 	function getMitigationsFile(req, res, next) {
-		TeamspaceSettings.getMitigationsStream(req.params.account).then((mitigations) => {
+		TeamspaceSettings.getMitigationsFile(req.params.account).then((mitigationsStream) => {
 			const timestamp = (new Date()).toLocaleString();
-			const filenamePrefix = (req.params.account + "_" + timestamp + "_").replace(/\W+/g, "_");
+			const filenamePrefix = (req.params.account + "_" + timestamp).replace(/\W+/g, "_");
```

Comment:
I think we still want the underscore otherwise you'd have no underscore before `mitigation.csv`? or am I missing something?